File: 1884-minimum-changes-to-make-alternating-binary-string/minimum-changes-to-make-alternating-binary-string.java
public class Solution {
    public int minOperations(String s) {
        int c = 0, x = 0;

        for (int i = 0; i < s.length(); i++) {
            if (i % 2 == 0 && s.charAt(i) == '0')
                c++;
            if (i % 2 == 1 && s.charAt(i) == '1')
                c++;

            if (i % 2 == 0 && s.charAt(i) == '1')
                x++;
            if (i % 2 == 1 && s.charAt(i) == '0')
                x++;
        }

        System.out.println(c + " " + x);

        return Math.min(c, x);
    }
}

File: 1884-minimum-changes-to-make-alternating-binary-string/minimum-changes-to-make-alternating-binary-string.cpp
class Solution {
public:
    int minOperations(string s) {
        int i,j,k,c=0,x=0;

        for(i=0;i<s.size();i++){
            if(i%2==0 and s[i]=='0')
            c++;
            if(i%2==1 and s[i]=='1')
            c++;

            if(i%2==0 and s[i]=='1')
            x++;
            if(i%2==1 and s[i]=='0')
            x++;
        }

        cout<<c<<" "<<x;

        return min(c,x);
    }
};